cfscrape/__init__.py

```diff
@@ -152,20 +153,26 @@ def solve_cf_challenge(self, resp, **original_kwargs):
         body = resp.text
         parsed_url = urlparse(resp.url)
         domain = parsed_url.netloc
-        submit_url = "%s://%s/cdn-cgi/l/chk_jschl" % (parsed_url.scheme, domain)
+        if self.org_method is None:
+            self.org_method = resp.request.method
+        submit_url = "%s://%s/%s" % (parsed_url.scheme,
+                                     domain,
+                                     re.findall('\<form id=\"challenge-form\" action=\"\/(.*)\?', resp.text)[0])
```

Comment:
I might be wrong here, but i feel doing the `[0]` at this point opens us up to another "unknown exception" if this changes at any point. It might be worth doing just the findall, and then if the length is 0 - throw a proper exception? Just a thought.

Reply:
0 is to take first match the re.findall take
if there is exception, them refindall did not find any thing

Reply:
... correct... But a proper error might want to be raised to indicate that something that was expected, is not there - and why/how. Standard index errors are not always clear within the context.
re: https://github.com/Anorov/cloudflare-scrape/blob/e4f31ed0fbc8aa5ba9ee8a75cf739d08575f1451/cfscrape/__init__.py#L82

I am just being picky though, feel free to continue without it.